rules/internal/objc_provider_utils.bzl

```diff
@@ -57,6 +57,8 @@ def _merge_dynamic_framework_providers(dynamic_framework_providers):
 
 def _merge_cc_info_providers(cc_info_providers, merge_keys = [
     "headers",
+    "includes",
```

Comment:
@thiagohmcruz do you happen to know if there are any implications of this change or other similar additions on build or IDE responsiveness?

Reply:
I didn't hit any issues while testing this change, if you have past context on this and are able to link issues / repro steps for something related to this I'm happy to investigate!